mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -233,7 +240,7 @@ CLASS ahl net/minecraft/entity/Entity
 	METHOD bE getMovementDirection ()Lex;
 	METHOD bF getComponentHoverEvent ()Ljl;
 	METHOD bG getBoundingBox ()Lcnf;
-	METHOD bI getEyeHeight ()F
+	METHOD bI getStandingEyeHeight ()F
```

Comment:
What is the difference between `getStandingEyeHeight` and `LivingEntity#getActiveEyeHeight`?

Reply:
the active eye height does not handle when the entity is sleeping; the standing eye height is precomputed when the entity is constructed